joplin/base/views/new_page_from_modal.py

```diff
@@ -93,6 +93,12 @@ def new_page_from_modal(request):
                     page=page,
                     permission_type='edit'
                 )
+                pvr = PageViewRestriction.objects.create(
+                    page=page,
+                    restriction_type=PageViewRestriction.GROUPS,
+                )
+                pvr.groups.set(department_group)
+
         else:
```

Comment:
https://www.laceyhenschel.com/blog/2018/8/1/search-history-adding-page-privacy-in-wagtail-programmatically

Reply:
this is interesting, the inclusion of a "public" makes me think these were originally intended for creating private parts of public django sites, but it seems like a great fit for our needs